cmd/kpod/push.go

```diff
@@ -29,6 +31,14 @@ var (
 			Name:  "cert-dir",
 			Usage: "`pathname` of a directory containing TLS certificates and keys",
 		},
+		cli.BoolFlag{
+			Name:  "compress",
+			Usage: "compress tarball image layers when saving to directory using the 'dir' transport. (default is same compression type as source)",
+		},
+		cli.StringFlag{
+			Name:  "format, f",
+			Usage: "manifest type (oci, v2s1, or v2s2) to use when saving image using the 'dir:' transport (default is manifest type of source)",
```

Comment:
saving? is push supposed to push to a registry?

Reply:
kpod push pushes to a registry and other transports as well.